Owncloud iOs Client/AppDelegate.m

```diff
@@ -1285,10 +1294,14 @@ - (void) restoreDownloadsInProccessFromSystemWithIdentificator: (NSString*)ident
     
     NSURLSessionConfiguration *configuration = nil;
     
-    if (IS_IOS8 || IS_IOS9) {
-        configuration = [NSURLSessionConfiguration backgroundSessionConfigurationWithIdentifier:identifier];
+    if (k_is_sso_active || !k_is_background_active) {
```

Comment:
In that cases the identifier came on the call of that method... so create a new method to do the same that make the method backgroundSessionConfigurationWithIdentifier it is redundant...